.travis.yml

```diff
@@ -37,6 +37,7 @@ install:
 before_script:
 - export TSLINT_FLAGS='-c ./tslint_ci.json'
 - export LONG_RUNNING_ENABLED=1
+- ./packages/iov-ethereum/startGanache.sh
```

Comment:
This is a good place to set ETH_ENV as well. When unset, no ethereum is available (like on Windows CI)